matlab_functions/structure/StructureDesignerRibWing.m

```diff
@@ -337,7 +337,7 @@
     %% inertia tensor recalculation
     Ixx=sum(Y.*Y*(m_distr/b*delta+wing_m_distr));
     for k=1:floor(n/2)
-        Ixx=Ixx+2*y(k*round(N/(n+1)))*y(k*round(N/(n+1)))*m_propulsion;
+        Ixx=Ixx+2*y(k*round(N/(n+1)))*y(k*round(N/(n+1)))*(m_propulsion/n); % Each unit's weight is Total divided by 'n'
```

Comment:
@philipoe 
I don't really get why that is the right code.
As you suggested, in case of 'n==3',
the code you gave would be : 
'y(k*round(N/(4)))*y(k*round(N/(4)))*(m_propulsion/1);'
But, actually the distributed mass is (m_propulsion/3). So I don't see any error in my code when n is odd number. Am I missing Anything??

Reply:
OK.

Looked at it again. I guess both are fine. For n=3, my code gives delta_Ixx=y(N/4)^2*m_propulsion = (1/2*b)^2*m_propulsion. This is OK because it also gives a lower Ixx than for n=2 which is correct because the center engine does not contribute to Ixx anymore (at least not through Steiner's law). Your code is however also physically correct. So yes, we can keep it as you proposed.

_Note for myself: Placing, in the case of n=2, the engines at 2/3*b is not what standard aerospace design would recommend. Change this to e.g. 1/3._